x11-misc/gpaste/gpaste-3.36.3.ebuild

```
@@ -0,0 +1,63 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION="0.30"
+VALA_USE_DEPEND="vapigen"
+
+inherit meson vala
+
+DESCRIPTION="Clipboard management system"
+HOMEPAGE="https://github.com/Keruspe/GPaste"
+SRC_URI="https://github.com/Keruspe/GPaste/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="introspection gnome vala"
+REQUIRED_USE="vala? ( introspection )"
+
+DEPEND="
+	vala? ( $(vala_depend) )
+	virtual/pkgconfig
+"
+RDEPEND="${DEPEND}
+	gnome? (
+		gnome-base/gnome-shell
+	)
+"
+BDEPEND="
+	dev-libs/appstream-glib
+	>=dev-libs/glib-2.48:2
+	>=dev-libs/gobject-introspection-1.48.0
+	sys-apps/dbus
+	>=x11-libs/gdk-pixbuf-2.34:2
+	>=x11-libs/gtk+-3.20:3
+	x11-libs/libX11
+	x11-libs/libXi
+	gnome? (
+		gnome-base/gnome-control-center:2
+		media-libs/clutter
+		x11-libs/pango
```

Your DEPEND and BDEPEND blocks are the other way around, so just move BDEPEND above RDEPEND and call it DEPEND. Then move your current DEPEND where BDEPEND is now, and call it BDEPEND.

```suggestion
DEPEND="
	dev-libs/appstream-glib
	>=dev-libs/glib-2.48:2
	>=dev-libs/gobject-introspection-1.48.0
	sys-apps/dbus
	>=x11-libs/gdk-pixbuf-2.34:2
	>=x11-libs/gtk+-3.20:3
	x11-libs/libX11
	x11-libs/libXi
	gnome? (
		gnome-base/gnome-control-center:2
		media-libs/clutter
		x11-libs/pango
	)
BDEPEND="
	vala? ( $(vala_depend) )
	virtual/pkgconfig
"
RDEPEND="${DEPEND}
	gnome? (
		gnome-base/gnome-shell
	)
"
```

Rest looks good.